src/KeyManager.ts

```diff
@@ -1,13 +1,454 @@
-import kbpgp from 'kbpgp'
+import fs from 'fs'
+import crypto from 'crypto'
+import chalk from 'chalk'
+import Path from 'path'
+import { promisify } from 'util'
 
-// we get back an object
-// not a series of modules I guess
-// otherwise the default is usually has to be import { default as ... } from ...
-// kbpgp.const.openpgp
-// so that's good
+
+
+// js imports
+const kbpgp = require('kbpgp')
+var F = kbpgp["const"].openpgp
+const zxcvbn = require('zxcvbn')
+
+// TODO: remember that the symmetric keys will just be stored in vault directories.
```

Comment:
Can you put these comments into the relevant issue. I don't really like having important notes in the code, especially if they haven't been reified to the code.

Reply:
One of these comments was Aashwin (I think?) talking about changing crypto libraries and making it able to support both asymm and symm encryption but I think we no longer want that, we are integrating kbpgp into our code and depending on asymmetric pubkey encryption.

The other comments I feel are already captured in existing issues.

There were also some TODO comments that seemed more like statements so I've take those out as well.